plugins/jobs/web_client/js/views/JobListWidget.js

```diff
@@ -85,6 +129,30 @@ girder.views.jobs_JobListWidget = girder.View.extend({
         window.setTimeout(function () {
             tr.removeClass('g-highlight');
         }, 1000);
+    },
+    _filterJobs: function (jobs) {
+        var filterJobs = [], self = this;
+
+        filterJobs = this.collection.filter(function (job) {
+            return ((_.isEmpty(self.typeFilter) || _.isUndefined(job.attributes.type) ||
```

Comment:
If I'm reading this right, the filter will include all jobs that are of the filtered set of types/status, and also those jobs that don't have a type or status?